src/login.html

```diff
@@ -8,7 +8,7 @@ <h1 style="margin-top:1em;text-align: left;">${HeaderPleaseSignIn}</h1>
             </div>
 
             <div class="inputContainer">
-                <input is="emby-input" type="text" id="txtManualName" required="required" label="${LabelUser}" autocomplete="username" />
+                <input is="emby-input" type="text" id="txtManualName" required="required" label="${LabelUser}" autocomplete="username" autocapitalize="off" />
```

Comment:
Perhaps we can also make this non autocorrectable as well?

```suggestion
                <input is="emby-input" type="text" id="txtManualName" required="required" label="${LabelUser}" autocomplete="username" autocapitalize="off" autocorrect="off" />
```

Reply:
We could add `autocapitalize="off"` to the password field right after, as well.

Reply:
As the password field is tagged as such, Android at lease defaults to no autocaptialization. Probably won't hurt but also not required I think.

Reply:
I tested and it seems that, with the ``autocomplete`` label set to username, Chrome doesn't autocorrect the field. It seems that ``autocomplete`` is the standard label to rule them all, so no specific labels for each situation (aside from autocapitalize). This means that passwords are not set to capitalize by default (as @MrTimscampi suggested to do as well) because it's implicit in the autocomplete tag. Usernames won't be autocorrected either because of that tag.

I approve the original commit. Thanks for your contribution! @rexbron